lib/multi_ruby_runner/version_manager.rb

```diff
@@ -7,9 +7,9 @@ class VersionManager
     def self.detect
       which_ruby = `which ruby`
       case which_ruby
-      when /\/\.rbenv\//
+      when /\/\.*rbenv\//
```

Comment:
Let's use a questionmark instead of asterisk. We're looking for zero or one, not zero or more periods.

```suggestion
      when /\/\.?rbenv\//
```